src/plugins/data/public/antlr/opensearch_ppl/grammar/OpenSearchPPLParser.g4

```diff
@@ -775,11 +775,11 @@ valueList
    ;
 
 qualifiedName
-   : ID // (DOT ident)* # identsAsQualifiedName
+   : ID // ident (DOT ident)* # identsAsQualifiedName
    ;
 
 tableQualifiedName
-   : ID // tableIdent (DOT ident)* # identsAsTableQualifiedName
+   : tableIdent (DOT ident)* # identsAsTableQualifiedName
```

Comment:
would issues from https://github.com/opensearch-project/OpenSearch-Dashboards/pull/8087#discussion_r1765952671 come back?

Reply:
The issue I mentioned in that comment is what the change I've done in `opensearch_ppl_autocomplete.ts` should handle, since we can prevent the grammar from causing the code completion engine to stay on that rule, thereby suggesting a table name twice.